scripts/shell/bt_nus_shell.py

```diff
@@ -1,84 +1,88 @@
-#!/usr/bin/env python
-# Copyright (c) 2019 Nordic Semiconductor ASA
+#!/usr/bin/env python3
+# Copyright (c) 2019-2020 Nordic Semiconductor ASA
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+# flake8: noqa
 
 import logging
-import Queue
+import queue
 import time
 import socket
 import sys
-from thread import *
+from _thread import *
 import argparse
 
-HOST = ''   # Symbolic name meaning all available interfaces
-PORT = 8889 # Arbitrary non-privileged port
+from pc_ble_driver_py import config
+config.__conn_ic_id__ = "nrf52"
+
+from nordicsemi.dfu.dfu_transport_ble import DFUAdapter
+from pc_ble_driver_py.exceptions import NordicSemiException
+from pc_ble_driver_py.ble_driver import Flasher, BLEDriver, BLEDriverObserver, BLEConfig, BLEConfigConnGatt,\
+    BLEUUIDBase, BLEUUID, BLEAdvData, BLEGapConnParams
+from pc_ble_driver_py.ble_driver import ATT_MTU_DEFAULT
+from pc_ble_driver_py.ble_adapter import BLEAdapter, BLEAdapterObserver, EvtSync
+
+
+CFG_TAG = 1
 
 parser = argparse.ArgumentParser(description='BLE serial daemon.')
 parser.add_argument('--com', help='COM port name (e.g. COM110).',
-                    nargs='*', required=False, default="")
+                    nargs='*', required=True, default="")
```

Comment:
Switching `--com` from `required=False` to `required=True` is a user visible change that is not related to Python 3 which is not explained in the commit log. Why is it in this commit? Why is the change being done?